js/config/app.js

```diff
@@ -9,7 +9,7 @@ define(function () {
   appConfig.cacheSources = false;
   appConfig.pollInterval = 60000;
 	appConfig.cohortComparisonResultsEnabled = false;
-	appConfig.userAuthenticationEnabled = false;
+	appConfig.userAuthenticationEnabled = true;
```

Comment:
Let's not change the default configuration for security

Reply:
You're right.
Fixed